assets/core/scss/_core.scss

```diff
@@ -305,14 +304,17 @@ textarea {
 */
 
 #main-area {
-  display: table-cell;
```

Comment:
I understand that Caniuse says it's supported but we had several problems even with simple flexbox layouts, especially in Safari both for Mac and iOS. That's why we never really use flexbox for critical elements like the base layout.